app/assets/stylesheets/sul-variables.scss

```diff
@@ -27,6 +27,7 @@ $stanford-black: #2e2d29;
 $teal: #00505c;
 $warm-grey: #3f3c30;
 $white: #ffffff;
+$content-advice-bg: #FEC51D1A;
```

Comment:
I think we should avoid creating new SASS variables and use CSS variables instead. Can we also call this `--stanford-illuminating-dark`?